src/renderers/webgl/WebGLBackground.js

```diff
@@ -108,12 +109,14 @@ function WebGLBackground( renderer, state, objects, premultipliedAlpha ) {
 			boxMesh.material.uniforms.flipEnvMap.value = texture.isCubeTexture ? - 1 : 1;
 
 			if ( currentBackground !== background ||
-			     currentBackgroundVersion !== texture.version ) {
+				currentBackgroundVersion !== texture.version ||
+				currentTonemapping != renderer.toneMapping ) {
```

Comment:
Maybe `!==` instead of `!=`?

Reply:
Oops. Thanks for the review. I'll fix it.